RecoParticleFlow/PFProducer/interface/PFBlockAlgo.h

```diff
@@ -153,11 +151,14 @@ class PFBlockAlgo {
 		    PFBlockLink::Type& linktype, 
 		    reco::PFBlock::LinkTest& linktest,
 		    double& dist) const;
-      
+  
+  std::auto_ptr< reco::PFBlockCollection >    blocksNew_;
   std::auto_ptr< reco::PFBlockCollection >    blocks_;
```

Comment:
`auto_ptr` has been deprecated. When adding new code, wouldn't it be better to use `shared_ptr` or `unique_ptr` instead?